ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```diff
@@ -14,5 +14,9 @@ ref:
       default: 'null'
       documentation: |-
         The instance type to use for compute nodes (e.g. AWS https://aws.amazon.com/ec2/instance-types/)
+    - name: ZONE
```

Comment:
Since we're parameterizing anyway we should probably just take zone as a number (i could see 1, 2, and 3 being all valuable, where 2 is the default).  When you make 2 the default, then you can just take the randomized list of zones and restrict it to a length (i.e. define the list of zones, then use bash to cut it to the appropriate length based on input).

Reply:
i don't think allowing configuration of the number of zones is a great idea,
- we probably can't fulfill that request in all regions and also not always due to limitations like quotas, outages etc.
  and if we can't fulfill the count that is probably going to cause problems in test assumptions.
- having more zones cost us more, it will be to much easier to set it too high when not absolutely required

so i like the single / multiple option because that's what most people want anyways...

Reply:
We probably should be testing 3 in some jobs because that's how OSD runs.  The default is 2 (cost).  A new use case here is 1 (testing).  Someone might legitimately need to test 4 zones (the product should absolutely work in a 4 zone config).  Right now we're focused on 1 and 2, and single/multiple is not the full set of configs we have.

Reply:
> Someone might legitimately need to test 4 zones (the product should absolutely work in a 4 zone config). 

if it works in 2 (multiple) it should work in 4, i don't really see how that adds any additional testing other than someone being overly pedantic. but 🤷🏽‍♂️